addons/Slideshow/src/presenter.js

```diff
@@ -1704,6 +1799,11 @@ function AddonSlideshow_create() {
             };
         }
 
+        var maxDuration = parseInt(model['Presentation duration']);
```

Comment:
Można użyć ModelValidationUtils.validateInteger.
Dodatkowo według MDN, na części przeglądarek, parseInt bez podania drugiego argumentu traktuje jako liczby w zapisie ósemkowym, więc jeżeli ktoś podałby 0 jako pierwszy znak mogłoby to się zachowywać inaczej na róznych przeglądarkach